sdk/ml/azure-ai-ml/azure/ai/ml/_ml_client.py

```diff
@@ -381,6 +382,13 @@ def __init__(
             **kwargs,
         )
 
+        self._service_client_10_2024_preview = ServiceClient102024Preview(
+            credential=self._credential,
+            subscription_id=self._operation_scope._subscription_id,
```

Comment:
If you want to have this PR merged in first, could you fix the `subscription_id` as Josh shared earlier.

Reply:
thanks Jing, yes please ref _service_client_07_2024_preview in `main` for any doubt

Reply:
addressed